backend/core/src/listings/dto/listing-published-create.dto.ts

```diff
@@ -102,6 +103,7 @@ export class ListingPublishedCreateDto extends OmitType(ListingCreateDto, [
 
   @Expose()
   @IsBoolean({ groups: [ValidationsGroupsEnum.default] })
+  @IsOptional({ groups: [ValidationsGroupsEnum.default] })
   referralOpportunity: boolean
```

Comment:
question: should this be marked as optional in the type def like `referralOpportunity?: boolean`